all_test.go

```diff
@@ -32,6 +32,11 @@ and Postgres backend. I'm not sure that testing without the race detector adds a
 */
 
 var backend_postgres = flag.Bool("backend_postgres", false, "Run tests against Postgres backend")
+var joeUserId UserId = 1
```

Comment:
I guess it's OK if we leave these hardcoded, but then at least let's leave a comment here saying `These hard-coded UserId values are predictable because we always drop & recreate the postgres backend when running tests, so we know that our IDs always start at 1`. Reading code with magic constants is very annoying if they aren't explained.